sed/binning.py

```diff
@@ -1,7 +1,6 @@
 # All functions in this file are adapted from https://github.com/mpes-kit/mpes
 from functools import reduce
 from typing import Sequence
-from typing import Tuple
```

Comment:
if we stick to Python 3.8, this import is needed

Reply:
Although `pyenv` makes it really easy to choose any Python version, we should stick not to the bleeding-edge ones. 
I chose 3.8 because it is the one installed on Maxwell.

Reply:
Do you use 3.8 features that are not part of 3.7? I would stick to the lowest version that provides the features necessary for the code. The old packages run just fine in 3.7. I would not break this compatibility without very good reason

Reply:
> Do you use 3.8 features that are not part of 3.7? I would stick to the lowest version that provides the features necessary for the code. The old packages run just fine in 3.7. I would not break this compatibility without very good reason

Currently not in use but the importlib.metadata can end up being quite useful in many situations. For example to use in single source versioning throughout the package. 
I am not aware if any other features from 3.8 are or will be used. With 3.10, I already had some reason to have used it.

Reply:
> Do you use 3.8 features that are not part of 3.7? I would stick to the lowest version that provides the features necessary for the code. The old packages run just fine in 3.7. I would not break this compatibility without very good reason

I chose 3.8 for two reasons:
- It's the default in Maxwell when you spawn a node
- Some dependencies (pandas, dask, ...) require, at least, python 3.8 in their last releases. As in some point, we will need to update it anyway, I decided to do it the sooner the better considering this library is still under heavy development

Reply:
I also agree that python 3.8 is the better choice. 
Earlier versions will only cause us to need to update sooner than later.
Bleeding edge versions like 3.10 are not easy to handle on the user side, outweighting the advantages posed by the new features.